src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        Random random = new Random();
+        Colours[] colour = Colours.values();
+        int randomIndex = random.nextInt(colour.length - 1);
+
+        return String.valueOf(colour[randomIndex]);
```

Comment:
Magic numbers should be avoided. In this case, 'colour.length - 1' contains a magic number. It would be better to define a constant for the array length or use it directly without subtraction.